pkg/services/playlist/playlistimpl/playlist.go

```diff
@@ -44,7 +44,31 @@ func (s *Service) GetItems(ctx context.Context, q *playlist.GetPlaylistItemsByUi
 }
 
 func (s *Service) Search(ctx context.Context, q *playlist.GetPlaylistsQuery) (playlist.Playlists, error) {
-	return s.store.List(ctx, q)
+	playlists, err := s.store.List(ctx, q)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: single call
+	if q.IncludeItems {
```

Comment:
NOTE -- the frontend does not ever call this, so no need for it to be fast.  We can have it, but also worth making it a clearly deprecated function